src/callbacks/callbacks.jl

```diff
@@ -434,6 +434,64 @@ function save_restart_func(integrator, output_dir)
     return nothing
 end
 
+Base.@kwdef mutable struct WallTimeEstimate
+    n_calls::Int = 0
+    n_next::Int = 1
+    t_last::Float64 = -1
+    ∑Δt_wall::Float64 = 0
```

Comment:
Can you please add docstrings to the various attributes describing what they are?

Also, maybe we want change `t_last` to `t_wall_last` to disambiguate it with the actual simulation time